package.json

```diff
@@ -129,7 +130,6 @@
     "artsy-backbone-mixins": "artsy/artsy-backbone-mixins",
     "artsy-eigen-web-association": "artsy/artsy-eigen-web-association",
     "artsy-ezel-components": "artsy/artsy-ezel-components",
-    "artsy-gemini-upload": "0.0.6",
```

Comment:
lol turns out there's [yet another library](https://github.com/artsy/artsy-gemini-upload) with a slightly different API...this one was a jQuery extension.

Reply:
This was maybe the last use of this library, according to [my search](https://github.com/search?q=org%3Aartsy+artsy-gemini-upload&type=Code). Can we mark that library as deprecated, archive the repo, and update the project list?

Reply:
Definitely!